apiv2/authentication.py

```diff
@@ -45,7 +45,12 @@ def authenticate(self, request):
         We override this method to check the status of related ApiV2Client.
         Check that ApiV2Client associatied to the given acess_token has not been disabled.
         """
-        super_response = super(OAuth2Authentication, self).authenticate(request)
+        try:
+            super_response = super(OAuth2Authentication, self).authenticate(request)
+        except UnicodeDecodeError:
+            # Catch excpetion when data can not be decoded
```

Comment:
one small thing here - it's great that we have a comment! However it would be nice to have this a little more explicit as to the reason, e.g.
```py
# If the request contains html entities that don't decode to valid UTF8, an exception is raised during oauth validation, even if it's on a field/parameter unrelated to oauth (#793)
# TODO: Check if this is still needed with Python3/oauthlib2
```